pkg/config/msp.go

```diff
@@ -700,9 +781,6 @@ func getOrgMSPValue(org *cb.ConfigGroup) (*cb.ConfigValue, error) {
 	return configValue, nil
 }
 
-// YEAR is a time duration for a standard 365 day year.
-const YEAR = 365 * 24 * time.Hour
-
 // RevokeCertificateFromMSP takes a variadic list of x509 certificates, creates
```

Comment:
Do we want to leave around this old version? Same question for AddRootCAToMSP.

Also, the new version (aka the one on ConfigTx) doesn't have the latest code for this function. Notably, it's missing the new validateCert logic.

Reply:
I think I left this after resolving some merge conflicts. I’ll fix it.

Reply:
🛠